chart/values.yaml

```diff
@@ -24,6 +24,17 @@ redis:
       cpu: 100m
       memory: 128Mi
 
+metrics:
+  enabled: false
+  serviceMonitor:
+    honorLabels: true
+    scrapeTimeout: 10s
+    relabelings:
+      - replacement: ots
+        targetLabel: application
+      - sourceLabels:
+          - pod
+        targetLabel: instance
```

Comment:
Hm why?

Reply:
to have proper naming in prometheus. Have to check if it's working

Reply:
Hmm not sure wheter it makes sense to provide a default relabeling config or to leave it to the operator installing the chart to provide those… 🤔